_rules/aria-attr-defined-5f99a7.md

```diff
@@ -71,7 +71,10 @@ This `div` element with a role of `alert` has an `aria-live` attribute which is
 This `input` element with a role of `spinbutton` has multiple `aria-*` attributes specified which are all defined in [WAI-ARIA Specifications][].
 
 ```html
-<input role="spinbutton" aria-valuemax="100" aria-valuemin="0" aria-valuenow="25" type="number" value="25" />
+<label
+	>Enter a number between 0 and 100:
```

Comment:
It seems add to have a new line in the middle of a tag. Minor typo, but should probably be fixed.

Reply:
That prettier being somewhat aggressive and cuting the line due to the long `input` element…
I've replaced this with a `label for=…` to make the result a bit more "logical".